src/bentoml/_internal/bento/bento.py

```diff
@@ -198,9 +200,16 @@ def create(
         for model in svc.models:
             models.add(model)
         # Add all models required by service runners
+        has_triton_runner = False
         for runner in svc.runners:
-            for model in runner.models:
-                models.add(model)
+            # only add models if given runner is not a TritonRunner
```

Comment:
Why do we need this? Are we specifying models in the triton runner?

Reply:
triton runners models is just a list of string, not actual model in the store atm.